oaknational/Oak-Web-Application
# Feat/storybook update

## Description

- update storybook to version 7

## Issue(s)

Fixes #1248 

## How to test

1. Go to storybook
2. Click on on the components
3. You should see all components functioning

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
